mmf/models/albef/vit.py

```diff
@@ -0,0 +1,515 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+# Initial version was taken from https://github.com/rwightman/pytorch-image-models
```

Comment:
We should try to avoid copy pasting so much code from `timm` library. It will be better to optionally import timm and just modify the necessary methods required in the vision transformer implementation for ALBEF model.

Reply:
@vedanuj thank you for your feedback. I will make sure to fix these in the next PR when I add the ALBEF model.